src/parse/parser_public.ts

```diff
@@ -625,6 +635,13 @@ export class Parser {
             ? config.dynamicTokensEnabled
             : DEFAULT_PARSER_CONFIG.dynamicTokensEnabled
 
+        this.avoidDynamicCodeDuringAnalysis = has(
```

Comment:
Perhaps it should be possible to overwrite this flag using a **global** env.
This will allow **end users** of a parser written using Chevrotain to modify
it without relying on the author of the specific parser.